cpp/open3d/geometry/PointCloud.cpp

```diff
@@ -478,6 +478,38 @@ std::shared_ptr<PointCloud> PointCloud::RandomDownSample(
     return SelectByIndex(indices);
 }
 
+std::shared_ptr<PointCloud> PointCloud::GetFarthestPointSample(
+        size_t num_samples) const {
+    if (num_samples <= 0) {
+        utility::LogError(
+                "[GetFarthestPointSample] Illegal number of points, the number "
+                "should larger than 0");
+    }
+    if (num_samples >= points_.size()) {
```

Comment:
Should we throw a wraning instead of error? because if `num_samples > points_size()`, we can still run the program and ouput the original `PointCloud`

Reply:
Error.